cibuildwheel/linux.py

```diff
@@ -41,6 +41,9 @@ def build(project_dir, output_dir, test_command, test_requires, test_extras, bef
         exit(2)
 
     python_configurations = get_python_configurations(build_selector)
+    if len(python_configurations) == 0:
+        raise ValueError("Empty list of configuration to build. Check 'CIBW_BUILD' and 'CIBW_SKIP' environment variables. " \
```

Comment:
Final details before this is ready to merge! :)

To keep terminology consistent, what about "No build identifiers are selected by `CIBW_BUILD` and `CIBW_SKIP`."? Just a suggestion, because 1) the docs also talk about "build identifiers", and 2) the user never enters a list, so "list of configurations to build" might be confusing? Up to you.
Personally, I'd say you can leave out "Check documentation if python version identifier do not change", because we're already giving deprecation warnings when changing something?

Reply:
Base on introduction `manylinux2010` I see that deprecation warning is replacing wrong value with proper. So this if will not be activated. But It is possible that someone will ignore it and when next release of `cibuildwheel` remove deprecation inform code the build will fail. And there will be no deprecation warning. 

I put new proposition of text in commit.

Reply:
Yes, isn't that what a deprecation is? Time for people to adapt?
Eh, anyway... guess it will be fine.